setup/src/Magento/Setup/Test/Unit/Module/ConfigGeneratorTest.php

```diff
@@ -5,24 +5,54 @@
  */
 namespace Magento\Setup\Test\Unit\Module;
 
+use Magento\Framework\App\DeploymentConfig;
+use Magento\Framework\Config\Data\ConfigData;
+use Magento\Framework\Config\Data\ConfigDataFactory;
 use Magento\Framework\Config\File\ConfigFilePool;
+use Magento\Framework\Math\Random;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
 use Magento\Setup\Model\ConfigGenerator;
 use Magento\Framework\Config\ConfigOptionsListConstants;
+use Magento\Setup\Model\CryptKeyGenerator;
+use PHPUnit\Framework\TestCase;
 
-class ConfigGeneratorTest extends \PHPUnit\Framework\TestCase
+class ConfigGeneratorTest extends TestCase
 {
     /**
      * @var ConfigGenerator
      */
     private $configGeneratorObject;
 
+    /**
+     * @SuppressWarnings(PHPMD.LongVariable)
```

Comment:
I believe such rule removed from Magento 2 PHPMD configuration. If it's just Codacy complains, simply ignore this and don't add suppressing annotations. Please remove both `@SuppressWarnings(PHPMD.LongVariable)` introduced.

Reply:
Okay, I will remove them. Can such Issues be hidden/ignored on Codacy?

Reply:
Don't know, you can simply ignore it overall. There are some other dumb rules like "never use `else`" :)